x/k6/go.mod

```diff
@@ -14,14 +14,15 @@ require (
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/grafana/pyroscope-go/godeltaprof v0.1.8 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0 // indirect
```

Comment:
I think you need to do a `go mod tidy` on this, as `github.com/grpc-ecosystem/go-grpc-middleware` is a direct dependency.

Also, we need to use `github.com/grpc-ecosystem/go-grpc-middleware/v2` instead, to avoid other dependency conflicts with `google.golang.org/grpc`.

Reply:
Should be good now!

Reply:
Used the make target, should be good for real :)

Reply:
Unfortunately the make target doesn't sync `x/k6`. See https://github.com/grafana/pyroscope-go/pull/142#issuecomment-2695786845 for next steps to get CI to go green. I'll open a separate PR to improve the make target for the future.